source/content/core-updates.md

```diff
@@ -13,13 +13,13 @@ integration: [--]
 
 This doc includes instructions to make core updates to WordPress and Drupal sites hosted on the Pantheon WebOps platform.
 
-## Drupal 9
+## Drupal (Latest Version)
 
-Drupal 9 sites on Pantheon use [Integrated Composer](/guides/integrated-composer) to allow one-click core updates through the Dashboard.
+As of Drupal 8.8, Drupal only supports managing depedencies via composer. You can do that without checking in the vendor directory by using Pantheon's [Integrated Composer](/guides/integrated-composer) to allow one-click core updates through the Dashboard. Site maintainers can combine these one-click updates with our [Autopilot](/guides/autopilot) service to automate site module updates.
```

Comment:
Drupal 10 *still* supports non-Composer-based sites. *Pantheon* only supports Composer for Drupal 9+. Drupal 8.8.0+ makes it easier / recommended to use Composer.

Reply:
I'm not sure that needs to be clarified in these docs.....

Reply:
![image](https://user-images.githubusercontent.com/87093053/213309519-f6ad5899-ad8e-4917-a33b-d48bdf045b0e.png)

Reply:
The AI has spoken.

Reply:
https://www.drupal.org/docs/extending-drupal/installing-modules#s-several-ways-to-add-modules

Reply:
We could make this correct and still strongly lead the customer towards Composer by adding a qualifier:

"In order to be able to use all available contributed modules, Drupal requires that you use Composer to manage your dependencies."

Reply:
updated text as per greg